frontend/src/App.jsx

```diff
@@ -12,6 +14,8 @@ function App() {
         <Route path="/register" element={<Register />} />
         <Route path="/" element={<HomeUser />} />
         <Route path="/scoring" element={<Scoring />} />
+        <Route path="/survey" element={<Survey />} />
+        <Route path="/survey/:categoryId/" element={<Question />} />
```

Comment:
Remove last `/` in path `/survey/:categoryId/`, it might produce problems in futur if you want to nest other routes inside this one.